packages/docusaurus/src/webpack/templates/ssr.html.template.ts

```diff
@@ -16,20 +16,18 @@ export default `
     <% }); %>
     <%~ it.headTags %>
     <% it.stylesheets.forEach((stylesheet) => { %>
+      <link rel="preload" href="<%= it.baseUrl %><%= stylesheet %>" as="style" />
       <link rel="stylesheet" href="<%= it.baseUrl %><%= stylesheet %>" />
     <% }); %>
     <% it.scripts.forEach((script) => { %>
-      <link rel="preload" href="<%= it.baseUrl %><%= script %>" as="script">
+      <script src="<%= it.baseUrl %><%= script %>" defer></script>
     <% }); %>
   </head>
   <body <%~ it.bodyAttributes %>>
     <%~ it.preBodyTags %>
     <div id="__docusaurus">
       <%~ it.appHtml %>
     </div>
-    <% it.scripts.forEach((script) => { %>
-      <script src="<%= it.baseUrl %><%= script %>"></script>
-    <% }); %>
```

Comment:
I'm not sure how it improves anything either.

The previous preload permitted to load the script eagerly. 

Now with defer it probably doesn't change much the behavior.

I'm not against this change as it looks cleaner and probably works, but I'm suspicious is would improve anything considering the scripts were already at the end?